typescript/hardhat/index.ts

```diff
@@ -1,20 +1,40 @@
+import { MultiProvider, TestChainNames } from '@abacus-network/sdk';
 import '@nomiclabs/hardhat-waffle';
+import { ethers } from 'ethers';
 import { extendEnvironment } from 'hardhat/config';
 import { lazyObject } from "hardhat/plugins";
-import { TestAbacusDeploy } from './src/TestAbacusDeploy'
-export { TestAbacusDeploy } from './src/TestAbacusDeploy'
-export { TestRouterDeploy } from './src/TestRouterDeploy'
-
-
-import "hardhat/types/runtime";
+import 'hardhat/types/runtime';
+import { TestCoreDeploy } from './src/TestCoreDeploy';
 
 declare module 'hardhat/types/runtime' {
```

Comment:
I'd expect to see `declare module` in a .d.ts file instead of a .ts file

Reply:
not exactly sure how this hardhat plugin works
because this PR doesn't introduce this, going to punt